API.md

```diff
@@ -659,7 +661,7 @@ const schema = {
     username: Joi.string().default(generateUsername),
     firstname: Joi.string(),
     lastname: Joi.string(),
-    created: Joi.date().default(Date.now, 'time of creation'),
+    created: Joi.date().default(() => new Date(), 'time of creation'),
```

Comment:
How is that wrong ?

Reply:
Date.now will populate the time that the process started.

Caused some massive bugs in some projects where we were scratching our heads about why the timestamps of hundreds of thousands of events were all clustered around a 3 second timespan. The root cause was finally detected when we tested event creation and it said it was two days old, and turns out that was the last time the process was restarted.

Reply:
This was the original version, which matches the current API:

```
module.exports = Joi
	.date()
	.default(Date.now(), 'Date when the trackingEvent occurred')
	// Keep as a Date object instead of converting to a string.
	.raw();
```

Reply:
OH. I see the problem. `Date.now()` vs `Date.now`.

Reply:
Looks like the change on this line could be reverted.